sds_data_manager/utils/stackbuilder.py

```diff
@@ -192,6 +193,40 @@ def build_sds(
         env=env,
     )
 
+    # I-ALiRT IOIS ECR
```

Comment:
Make a separate `build_ialirt` function so it can be deployed independently?

Although you are using the SDS VPC, so maybe that is not possible...